spec/core/evaluation_spec.sh

```diff
@@ -17,6 +17,15 @@ Describe "core/evaluation.sh"
     End
```

Comment:
CodeFactor check fails for this file:  
```
spec/core/evaluation_spec.sh#L399
This is interpreted as a command name containing '='. Bad assignment or comparison? (SC2276)
```
https://github.com/shellspec/shellspec/pull/237/checks?check_run_id=3269637837

I only copied code, did not add anything particularly new.

**We could just ignore SC2276 like SC2016 is already ignored in this file if this is not actually an issue** 
( [line 1](https://github.com/shellspec/shellspec/pull/237/files#diff-d6c44027553744690864be9c389fd56b9e9f278d7be821c57ef8b26268b9842eL1) )

Reply:
It's an alias of `%putsn`, see [README.md](https://github.com/shellspec/shellspec#puts---putsn----output-a-string-with-newline).

So, definitely a false positive. 

Will add a suppression :mag:

Reply:
:information_source: c122bb8463f5667f3279c6abd7b6e2f45c430f7d

Reply:
CodeFactor seems to be using the latest shellcheck 0.7.2.
https://support.codefactor.io/i24-analysis-tools-open-source

I just updated ShellCheck on my pc and I get more warnings... 😧
Well, anyway, you don't have to worry about it.